### PR TITLE
Add EEP-00: Governance Model and Code of Conduct

### PR DESCRIPTION
**Pull request by segsell**
In this PR, we add `EEP-00 -- Governance Model and Code of Conduct`.

### CODE REVIEW

**Comment by janosg** [comment]
EEP-00 is ready. It proposes a code of conduct and formalizes the estimagic governance model. 

The **code of conduct** is very similar to the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) which is adopted by many numfocus affiliated projects. They have an excellent [FAQ](https://www.contributor-covenant.org/faq/) which explains why a code of conduct is needed. 

The **governance model** is very lightweight and formalizes three things: 1) Most of the work will continue to be done in the same pull request workflow as before. 2) Any interested party can object to changes in which case the change has to go through an Estimagic Enhancement Proposal process. 3) The steering council facilitates discussions, organizes events and serves the estimagic community in any way necessary. Its members do not have any privileges. 

All interested users can comment below with approval or objections. Objections need to come with clear explanations and proposals for improvement. If there are no objections until October 10, the EEP is automatically adopted. On Wednesday, October 5 at 6 pm there will be a public zoom meeting (link on zulip) on which the members of the first steering council will be elected.

File: docs/source/development/eep-00-governance-model.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+design should be discussed in an Estimagic Enhancement Proposal before the actual
+work starts. When the work is finished, the author of a pull
+request can request a review. In most cases, previous discussions will show who is a
+suitable reviewer. If in doubt, tag [janosg](https://github.com/janosg). Pull requests
+can be merged if there is at least one approving review.
+
+Reviewers should be polite, welcoming and helpful to the author of the pull request
+who might have spent many hours working on the changes. Major points should be discussed
+publicly on GitHub, but very critical feedback or small details can be moved to private
+discussions. Video calls can help if a discussion gets stuck. Keep in mind that many
```
Comment (hmgaudecker) [inline docs/source/development/eep-00-governance-model.md:74]:
```suggestion
discussions — if the latter are necessary at all (see
[the bottom section of this blog post](https://rgommers.github.io/2019/06/the-cost-of-an-open-source-contribution/)
for an excellent discussion of the burden that review comments place on
maintainers, which might not always be obvious).
Video calls can help if a discussion gets stuck. 
```

Reference is great, but should be broader than just the point that many people get emails IMO.

File: docs/source/development/eep-00-governance-model.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+people get notified on every review comment (see [here](https://rgommers.github.io/2019/06/the-cost-of-an-open-source-contribution/) for an execellent discussion).
```
Comment (hmgaudecker) [inline docs/source/development/eep-00-governance-model.md:75]:
```suggestion
```

File: docs/source/development/eep-00-governance-model.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+The code of conduct applies to all interactions related to code reviews.
+
+### Estimagic Enhancement Proposals (EEPs) / Controversial decision process
```
Comment (hmgaudecker) [inline docs/source/development/eep-00-governance-model.md:78]: If the convention is to have "estimagic" always in lowercase, I do not think we should capitalize in titles or as first word of a sentence, either. But no strong opinion. Should just be consistent, e.g. line 80 is lowercase in the middle of some text but line 109 capitalised.

File: docs/source/development/eep-00-governance-model.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Large changes to estimagic can be proposed in Estimagic Enhancement Proposals, short
+EEPs. They serve the purpose of summarising discussions
```
Comment (hmgaudecker) [inline docs/source/development/eep-00-governance-model.md:81]: Just a reminder to run automatic line-wrapping before merging.

File: docs/source/development/eep-00-governance-model.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+a formal process.
+
+EEPs are written as markdown documents that become part of the documentation. Opening
+an EEP means opening a pull request that adds the markdown document to the documentation.
+It is not necessary to already have a working implementations for the planned changes,
+even though it might be a good idea to have rough prototypes for solutions to the most
+challenging parts.
+
+If the author of an EEP feels that it is ready to be accepted they need to make a
+post in our [zulip workspace](https://ose.zulipchat.com) and a comment on the PR that
```
Comment (hmgaudecker) [inline docs/source/development/eep-00-governance-model.md:93]:
```suggestion
post in the relevant [Zulip topic](https://ose.zulipchat.com/#narrow/stream/221432-estimagic/topic/EEPs) and a comment on the PR that
```

File: docs/source/development/eep-00-governance-model.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+of all community members.
+- Develop a roadmap, break down large changes into smaller projects and find contributors
+to work on the implementation of these projects.
+- Ensure that new contributors are onboarded and assisted and that pull requests are
+reviewed in a timely fashion.
+- Step in as moderators when discussions get heated, help to achieve consensus on
+controversial topics and enforce the code of conduct.
+
+Current memebers of the Estimagic Steering Council are:
+[Janoś Gabler](https://github.com/janosg), [Annica Gehlen](https://github.com/amageh), [Hans-Martin von Gaudecker](https://github.com/hmgaudecker), [Tim Mensinger](https://github.com/timmens) and [Mariam Petrosyan](https://github.com/mpetrosian).
```
Comment (hmgaudecker) [inline docs/source/development/eep-00-governance-model.md:125]: Order alphabetically? Maybe a bulleted list?

**Review by hmgaudecker — COMMENTED** [review]
Looks great, just some suggestions for textual changes!

File: docs/source/development/eep-00-governance-model.md (hunk 6 above)
Comment (janosg) [reply docs/source/development/eep-00-governance-model.md:128]: Done.

File: docs/source/development/eep-00-governance-model.md (hunk 4 above)
Comment (janosg) [reply docs/source/development/eep-00-governance-model.md:84]: Done.

**Review by janosg — APPROVED** [review]

**Comment by segsell** [comment]
Looks great!!

File: docs/source/development/changes.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 (changes)=
 # Changes
-```{eval-rst}
-.. include:: ../../../CHANGES.md
+
+```{include} ../../../CHANGES.md
```
Comment (amageh) [inline docs/source/development/changes.md:4]: This makes the **Changes** header appear twice on the site.

File: docs/source/development/eep-00-governance-model.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+|            | `Janoś Gabler <https://github.com/janosg>`_,                     |
+|            | `Hans-Martin von Gaudecker <https://github.com/hmgaudecker>`_,   |
+|            | `Annica Gehlen <https://github.com/amageh>`_,                    |
+|            | `Sebastian Gsell <https://github.com/segsell>`_,                 |
+|            | `Tim Mensinger <https://github.com/timmens>_,                    |
+|            | `Mariam Petrosyan <https://github.com/mpetrosian>`_,             |
+|            | `Tobias Raabe <https://github.com/tobiasraabe>`_,                |
+|            | `Klara Röhrl <https://github.com/roecla>`_                       |
++------------+------------------------------------------------------------------+
+| Status     | Draft                                                            |
```
Comment (amageh) [inline docs/source/development/eep-00-governance-model.md:18]: Wouldn't we need to change the status here once adopted?

**Comment by mpetrosian** [comment]
Looks great! Thanks for putting it together!

File: docs/source/development/eeps.md [hunk 9]
```diff
@@ -1,28 +1,14 @@
...
+Estimagic Enhancement Proposals (EEPs) can be used to discuss and design large
+changes. EEP-00 details the EEP process, the estimagic governance model and the
+estimagic Code of Conduct. It is the only EEP that gets continuously updated.
 
 These EEPs are currently in place:
 
 ```{toctree}
 :maxdepth: 1
 
+eep-00-governance-model.md
```
Comment (amageh) [inline docs/source/development/eeps.md:12]: I'm not sure it is mentioned anywhere, where the code of conduct will be featured once the EEP is adopted? I suggest moving it from the EEP's to the Development section in the documentation.

**Review by amageh — APPROVED** [review]
Thank you for your work on this. Looks great!

**Review by timmens — APPROVED** [review]
I'm glad we formalized both our governance model and created a code of conduct. This is an important step for estimagic and was long overdue. I also like that we chose a slightly more restrictive code of conduct in comparison to the contributor covenant version. Hopefully, we will continue to see a vibrant community of contributors that can now rely on a formalized model of responsibilities as well as an environment that cares about their safety.

File: docs/source/development/eep-00-governance-model.md (hunk 8 above)
Comment (janosg) [reply docs/source/development/eep-00-governance-model.md]: Done.

File: docs/source/development/changes.md (hunk 7 above)
Comment (janosg) [reply docs/source/development/changes.md:2]: Done.

File: docs/source/development/eeps.md (hunk 9 above)
Comment (janosg) [reply docs/source/development/eeps.md:12]: Very good point! I added it as first item in the development section.

**Pull request merged**